tests/scale/utils.py

```diff
@@ -0,0 +1,63 @@
+from __future__ import annotations
+
+import time
+from contextlib import contextmanager
+from typing import Any, Callable, Generator
+
+import pytest
+import yaml
+from ocp_resources.machine_config_pool import MachineConfigPool
+from ocp_resources.resource import ResourceEditor
+
+
+def get_user_kubeconfig_context(kubeconfig_filename: str, username: str) -> str:  # skip-unused-code
```

Comment:
why is this function needed? can you please provide some context via a doscrting?

Reply:
done